xhr.bs

```diff
@@ -949,31 +945,24 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
    <li><p>If <var>processedResponse</var> is false, then set <a>this</a>'s <a>timed out flag</a> and
    <a for=fetch>terminate</a> <a for=/>fetching</a>.
 
-   <li><p>Run <a>handle response end-of-body</a> for <a>this</a> and <var>response</var>.
+   <li><p>Run <a>handle response end-of-body</a> for <a>this</a>.
   </ol>
 </ol>
 
 <p id=handle-response-end-of-file>To <dfn>handle response end-of-body</dfn> for an
-{{XMLHttpRequest}} object <var>xhr</var> and a <a for=/>response</a> <var>response</var>, run these
-steps:
+{{XMLHttpRequest}} object <var>xhr</var>, run these steps:
 
 <ol>
- <li><p>If <var>xhr</var>'s <a>synchronous flag</a> is set, then set <var>xhr</var>'s
- <a for=XMLHttpRequest>response</a> to <var>response</var>.
-
- <li><p><a>Handle errors</a> for <var>xhr</var> and <var>response</var>.
+ <li><p><a>Handle errors</a> for <var>xhr</var>.
```

Comment:
I expect some branch and return around here.

Reply:
Handle errors sets it to a network error and the next step returns in that case, right?

Reply:
Ah I see, thanks.

How about the normal case? Is it OK to call `handle errors` unconditionally?

Reply:
You're right, handle errors should keep the is network error conditional at the end and instead we should set response to a network error before invoking it.

Reply:
I think this should be addressed now.